### ISSUE TITLE
可以用于梯度筛选吗？

### ISSUE THREAD

**Issue by jerehrong** [PengtaoJiang/LayerCAM#12]
你好！我在调试您的代码的时候我本没有发现layercam.py文件中的self.gradients是在什么时候产生的。在运行完        logit.backward(gradient=one_hot_output, retain_graph=True)
        activations = self.activations['value'].clone().detach()这两行代码时监视器中self.gradients就有值了，我没有发现它时如何赋值的。
